README.md

```diff
@@ -364,6 +364,7 @@ List of projects that provide terminal user interfaces
 - [invidtui](https://github.com/darkhz/invidtui) A TUI Invidious client for Windows, Linux and MacOS, that fetches and plays audio/video from an invidious instance. Supports viewing and playing from playlists and channels as well.
 - [kew](https://github.com/ravachol/kew) A terminal music player for Linux
 - [line](https://github.com/pd3v/line) Tiny command-line midi sequencer and language for live coding
+- [marstui-audio](https://github.com/schooldanlp6/marstui-rustio) A nice audio management Interface, similar to pavucontrol with the benifit of customizing everything
```

Comment:
`benefit`

Reply:
the benifit is it is pavucontrol but as a tui -> this is good. (well It can technically also do a lot more like skip songs stop etc.). and it is customizable,

Reply:
I think he was mentioning that benefit is spelled wrong